test/v2/controllers/DeletePropertyAnnualSubmissionControllerSpec.scala

```diff
@@ -16,39 +16,36 @@
 
 package v2.controllers
 
-import api.controllers.ControllerBaseSpec
+import api.controllers.{ControllerBaseSpec, ControllerTestRunner}
 import api.mocks.services.{MockAuditService, MockEnrolmentsAuthService, MockMtdIdLookupService}
 import api.mocks.MockIdGenerator
-import play.api.libs.json.Json
-import play.api.mvc.Result
-import uk.gov.hmrc.http.HeaderCarrier
-import v2.mocks.requestParsers.MockDeletePropertyAnnualSubmissionRequestParser
-import v2.mocks.services.MockDeletePropertyAnnualSubmissionService
-import api.models.audit.{AuditError, AuditEvent, AuditResponse, GenericAuditDetail}
+import api.models.audit.{AuditEvent, AuditResponse, GenericAuditDetail}
 import api.models.domain.{Nino, TaxYear}
 import api.models.errors._
 import api.models.outcomes.ResponseWrapper
+import play.api.libs.json.{Json, JsValue}
+import play.api.mvc.Result
+import v2.mocks.requestParsers.MockDeletePropertyAnnualSubmissionRequestParser
+import v2.mocks.services.MockDeletePropertyAnnualSubmissionService
 import v2.models.request.deletePropertyAnnualSubmission._
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 class DeletePropertyAnnualSubmissionControllerSpec
     extends ControllerBaseSpec
+    with ControllerTestRunner
     with MockEnrolmentsAuthService
     with MockMtdIdLookupService
     with MockDeletePropertyAnnualSubmissionService
     with MockDeletePropertyAnnualSubmissionRequestParser
     with MockAuditService
     with MockIdGenerator {
 
-  private val nino          = "AA123456A"
-  private val businessId    = "XAIS12345678910"
-  private val taxYear       = "2023-24"
-  private val correlationId = "X-123"
+  private val businessId = "XAIS12345678910"
+  private val taxYear    = "2023-24"
 
-  trait Test {
-    val hc: HeaderCarrier = HeaderCarrier()
+  trait Test extends ControllerTest {
 
     val controller = new DeletePropertyAnnualSubmissionController(
```

Comment:
Can we move the Test trait to the bottom of the class so as to be consistent with the other controller specs (it also makes the files more readable since i can read the test as soon as I open the file and if I need specific details then I can scroll down to the Test trait to find those details)